README.md

```diff
@@ -142,11 +142,13 @@ The following is a brief explanation of each available command.
   CONTAINER_TIMEOUT_SECONDS=1800 codeclimate analyze // 30 minutes
   ```
 
+  Note: CONTAINER_TIMEOUT_SECONDS is not currently honored in the CLI.
```

Comment:
@GordonDiggs Hi!  Thanks, I hope I'm doing something wrong but support emailed me that the CLI doesn't allow increasing the engine timeout... I would love if it did work in some way...

I'm seeing this:

`error: (CC::Analyzer::Engine::EngineTimeout) engine duplication:stable ran for 900 seconds and was killed`

after `export CONTAINER_TIMEOUT_SECONDS=3600; codeclimate analyze`

My repo is open source so I'd love it if someone can show me how to get that to work.

https://codeclimate.com/github/jrafanie/manageiq